frontend/src/lib/MlmdUtils.test.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  Api,
+  Context,
+  GetContextByTypeAndNameRequest,
+  GetContextByTypeAndNameResponse,
+} from '@kubeflow/frontend';
+import { expectWarnings, testBestPractices } from 'src/TestUtils';
+import { Workflow, WorkflowSpec, WorkflowStatus } from 'third_party/argo-ui/argo_template';
+import { getRunContext } from './MlmdUtils';
+
+testBestPractices();
```

Comment:
It is great to enforce test best practices here. However, should we rename it from `testBestPractices` to `resetTestMock`? Since there are so many best practices on testing, it might not be clear to developers on when and how to use this function in test files.

Reply:
I'd suggest always using it, does that clear your confusion? We couldn't enforce that right now, because there are old tests that didn't follow the best practices.

Also, I plan to add more to the function, right now it's just a start.